### PR TITLE
feat: Update to use consistent font-family

### PR DESCRIPTION
**Pull request by jgphilpott**
**Linked to issue #204**

**Summary**
Okay, I think I have managed to do a full update of the font stack. I basically just did a repo wide search for the words 'font' and 'font-family' and updated everything that I found. Let me know if I missed something.

**Updates**
I updated the font stack from `Open Sans, Roboto, system-ui, sans-serif` to `Segoe UI, Ubuntu, sans-serif` and removed imports for `Open Sans` because its not being used anymore.

### CODE REVIEW

File: src/demo/index.php [hunk 1]
```diff
@@ -20,7 +20,6 @@ function gtag() {
     </script>
     <title>GitHub Readme Streak Stats Demo</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (DenverCoder1) [inline src/demo/index.php:22]: If not using Google Fonts on the demo site, this can be removed
```suggestion
```

**Review by DenverCoder1 — COMMENTED** [review]
Looks like this will work great. :+1: 

I don't think you missed anything, but it seems like the fonts on the demo site were also changed.

I didn't expect that it would be necessary to change it there, but I do think it's fine if the website uses the new font stack instead of Open Sans, so I wouldn't mind keeping it as is.

File: src/demo/index.php (hunk 1 above)
Comment (jgphilpott) [reply src/demo/index.php:22]: Okay looks good!

**Comment by DenverCoder1** [comment]
Looks great! Thanks for the contribution :tada:

**Comment by jgphilpott** [comment]
> Looks great! Thanks for the contribution tada

Awesome, my pleasure!

**Pull request merged**